charts/library/common/templates/classes/_service.tpl

```diff
@@ -30,6 +30,9 @@ metadata:
   {{- if eq ( $primaryPort.protocol | default "" ) "HTTPS" }}
     traefik.ingress.kubernetes.io/service.serversscheme: https
   {{- end }}
+  {{- if eq ( $svcType | default "" ) "LoadBalancer" }}
```

Comment:
I like the use of default here as a fallback to prevent this from breaking the app completely, my compliments!